### ISSUE TITLE
76/112 themes do not use filter |raw

### ISSUE THREAD

**Issue by pamtbaau** [getgrav/grav#3338]
After again a post on the forum from someone bumping into a theme which does not use filter `|raw`, I decided to run a quick test on all themes.

It seems that 77 themes out of 112 do not use filter `|raw` on `{{ *.content }}`, `{{ *css() }}`, or `{{ *js() }}`. 

Is there some way to:
- Notify developers that their theme needs to be updated?
- Notify users that these themes are outdated and will not work correctly without intervention?
- Add a warning banner to these themes in the theme repository?

**Comment by mahagr** [comment]
They can still be used, though. Those themes just need some configuration changes to work.

**Comment by pamtbaau** [comment]
True, they can be used with some config changes, however, many users picking a theme are new users who are not aware of auto-escaping and have no clue how to fix it.

Or do we want to:
- warn the user upfront about the compatibility issue
- offer the steps how to fix it
- and explain why the fix is a security risk

**Comment by msoler75** [comment]
To fix themes I put |raw in all lines like:

{{ page.content|raw }}
{{ assets.*|raw }}
